.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Suggestion (non-blocking): If we're adding Prettier, should we add this to the `pretest` npm script along with the ESLint check?